xdist/dsession.py

```diff
@@ -791,3 +366,6 @@ def pytest_testnodedown(self, node, error):
     # def pytest_xdist_rsyncfinish(self, source, gateways):
     #    targets = ", ".join(["[%s]" % gw.id for gw in gateways])
     #    self.write_line("rsyncfinish: %s -> %s" %(source, targets))
+
+
+__all__ = ['DSession']
```

Comment:
I agree, I don't think xdist is mature enough to promise any API other than what we promise through hooks.